core/opds2_import.py

```diff
@@ -514,14 +515,17 @@ def _extract_media_types_and_drm_scheme_from_link(
                     nested_acquisition = first_or_default(nested_acquisition.child)
                 [drm_type, media_type] = nested_types[-2:]
 
+                # We then check this returned pair of content types to make sure they match known
+                # book or audiobook and DRM types. If they do not match known types, then we skip
+                # the item.
```

Comment:
Are we skipping the `item` here or just this link? The link might be what you mean here, but `item` might be misconstrued to mean book, rather than link.